src/plugins/mobile/mobile.css

```diff
@@ -0,0 +1,94 @@
+@keyframes fadeAndScale {
+	0% {
+		opacity: 0
+	}
+
+	25% {
+		opacity: 1;
+	}
+
+	100% {
+		opacity: 0;
+	}
+}
+
+.video-js.v-mobile-ui {
```

Comment:
What is this class? There's no dependency between Vlitejs and Video.js, and I prefer it that way.

Reply:
I simply copy-paste, the class needs to be rewritten/adapted